frontend/src/components/ui-kit/components/Paginate/Paginate.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { memo } from 'react';
+import { Pagination, PaginationItem } from '@mui/material';
+import styles from './Paginate.module.scss';
+
+const Paginate = memo(({ pageCount, page, setPage }) => {
+  const handleChange = (e, value) => {
+    setPage(value);
+  };
+  return (
+    <div>
+      <Pagination
+        className={styles['pagination']}
+        count={pageCount}
+        siblingCount={0}
+        hidePrevButton={page <= 1}
+        hideNextButton={page >= pageCount}
+        page={page}
+        onChange={handleChange}
+        renderItem={(item) => (
+          <PaginationItem
+            {...item}
+            className={
+              item.type === 'previous'
```

Comment:
They don't write this way at all, very rarely you can use a double ternary expression and it's not desirable, the code becomes unreadable, you have if..else or switch